### PR TITLE
Updates and images for plugins documentation

### PR DESCRIPTION
**Pull request by PatrickOgle**
Screengrabs were added to the Specviz Plugins documentation.    Some minor edits were made and 
:guilabel: markdown added.

### CODE REVIEW

**Comment by PatrickOgle** [comment]
Ricky, can you please check  this statement:
"The statistics returned include the line centroid, width, peak flux, total flux, and equivalent width."
and this statement:
"Currently, it is assumed that the continuum has already been
normalized to unity."
for the Line Analysis Plugin documentation?

**Comment by PatrickOgle** [comment]
@javerbukh  @rosteen One other question regarding the Unit Conversion Plugin:

"The Spectrum1D object to be
converted is selected in the spectrum viewer :guilabel:`gear-->Data` tab.
It is not currently possible to select the data within the Plugin."

Is this true, and what is the reason for that, if so?

File: docs/specviz/plugins.rst [hunk 1]
```diff
@@ -3,102 +3,120 @@ Data Analysis Plugins
...
 =============
 
 The Line Analysis plugin returns statistics for a single emission
-line.  The emission line is selected by using the region tool in
+line.
+The emission line is selected via the :guilabel:`region` tool in
 the spectrum viewer to select a spectral subset.
 
 The statistics returned include the line centroid, width, peak flux,
 total flux, and equivalent width.
```
Comment (rosteen) [inline docs/specviz/plugins.rst:115]:
```suggestion
The statistics returned include the line centroid, gaussian sigma width, gaussian FWHM,
total flux, and equivalent width.
```

**Comment by PatrickOgle** [comment]
Doesn't Unit Conversion actually create a new Spectrum 1D object, with the new units anyway?
I see the new data object in the Data list.

**Comment by javerbukh** [comment]
Good point! I certainly think Unit Conversion could go through a refactor the next time we have availability.

**Pull request merged**